src/test/crimson/test_errorator.cc

```diff
@@ -18,6 +19,15 @@ struct errorator_test_t : public seastar_test_suite_t {
       }
     });
   }
+  ertr::future<> test_parallel_for_each() {
+    std::vector<int> loops = {1, 2, 3};
```

Comment:
this is probably not the best way to demonstrate the usage. as `loops` is captured by reference. so dereferencing it in reactor might lead to undefined behavior. i'd suggest either use
```c++
  static constexpr int SIZE = 42;
  ertr::future<> test_parallel_for_each() {
    auto sum = std::make_unique<int>(0);
    return ertr::parallel_for_each(
      boost::make_counting_iterator(0),
      boost::make_counting_iterator(SIZE),
      [sum=sum.get()](int i) {
        *sum += i;
      })
    .safe_then([sum=std::move(sum)] {
      int expected = std::accumulate(boost::make_counting_iterator(0),
                                     boost::make_counting_iterator(SIZE),
                                     0);
      ASSERT_EQ(*sum, expected);
    });
  }
```